spec/model/compatibility-check/check-model.spec.ts

```diff
@@ -1188,6 +1188,132 @@ describe('checkModel', () => {
             expectToBeValid(result);
         });
     });
+
+    describe('@calcMutations', () => {
+        it('rejects a missing @calcMutations with one operator', () => {
+            const result = run(
+                gql`
+                    type Test @rootEntity @modules(in: "module1", includeAllFields: true) {
+                        test: Int @calcMutations(operators: ADD)
+                    }
+                `,
+                gql`
+                    type Test @rootEntity {
+                        test: Int
+                    }
+                `,
+            );
+            expectSingleCompatibilityIssue(
+                result,
+                'Field "Test.test" should be decorated with @calcMutations(operators: [ADD]) (required by module "module1").',
+            );
+        });
+
+        it('rejects a missing @calcMutations with two operators', () => {
+            const result = run(
+                gql`
+                    type Test @rootEntity @modules(in: "module1", includeAllFields: true) {
+                        test: Int @calcMutations(operators: [ADD, MULTIPLY])
+                    }
+                `,
+                gql`
+                    type Test @rootEntity {
+                        test: Int
+                    }
+                `,
+            );
+            expectSingleCompatibilityIssue(
+                result,
+                'Field "Test.test" should be decorated with @calcMutations(operators: [ADD, MULTIPLY]) (required by module "module1").',
+            );
+        });
+
+        it('rejects one missing operator', () => {
+            const result = run(
+                gql`
+                    type Test @rootEntity @modules(in: "module1", includeAllFields: true) {
+                        test: Int @calcMutations(operators: [ADD, MULTIPLY])
+                    }
+                `,
+                gql`
+                    type Test @rootEntity {
+                        test: Int @calcMutations(operators: ADD)
+                    }
+                `,
+            );
+            expectSingleCompatibilityIssue(
+                result,
+                'Operator MULTIPLY is missing (required by module "module1").',
```

Comment:
Similar on the other validation messages. I omit the suggestion from now on. If you agree with incorporating the  field and type name for directive messages you can apply it for the others to.

```suggestion
                '@calcMutations on "Test.test" is lacking Operator MULTIPLY (required by module "module1").',
```